assets/lokomotive-kubernetes/packet/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -266,11 +266,13 @@ storage:
           :FORWARD ACCEPT [0:0]
           :OUTPUT ACCEPT [0:0]
           -A INPUT -i lo -j ACCEPT
-          -A INPUT -p tcp --dport 22 -j ACCEPT
-          -A INPUT -p tcp --dport 179 -j ACCEPT
-          -A INPUT -p tcp --dport 2379 -j ACCEPT
-          -A INPUT -p tcp --dport 2380 -j ACCEPT
-          -A INPUT -p tcp --dport 2381 -j ACCEPT
+          %{ for management_cidr in management_cidrs ~}
+          -A INPUT -s ${management_cidr} -p tcp --dport 22 -j ACCEPT
+          %{~ endfor }
+          -A INPUT -s ${node_private_cidr} -p tcp --dport 179 -j ACCEPT
+          -A INPUT -s ${node_private_cidr} -p tcp --dport 2379 -j ACCEPT
+          -A INPUT -s ${node_private_cidr} -p tcp --dport 2380 -j ACCEPT
+          -A INPUT -s ${node_private_cidr} -p tcp --dport 2381 -j ACCEPT
```

Comment:
Not sure if we should use the `node_private_cidr` or just `10.0.0.0/8`. I like the simplicity of just using `10.0.0.0/8`, but... no strong opinion.

What do you think?

Reply:
`node_private_cidr` could default to `10.0.0.0/8` maybe?

Reply:
Now that I think about it, this has been discussed in other issues, I think. The problem is that these rules can't be changed... so, it's complicated. I think both are fine for me (`10.0.0.0/8` vs `node_private_cidr`). Do whatever you consider best :)

Reply:
Decided to use `10.0.0.0/8`. Seems like a sane bootstrap value.